src/contentChunk.js

```diff
@@ -61,3 +61,29 @@ exports.extractStringPayloadIntoChunks = (payload) => {
     stream.end(payload)
   })
 }
+
+export const retrievePayload = (fileId, callback) => {
+  const db = connectionDefault.client.db()
+  if (!db) {
+      const err = new Error(`Payload retrieval failed. Database handle: ${db} is invalid`)
+      return callback(err)
+  }
+
+  if (!fileId) {
+    const err = new Error(`Payload retrieval failed: Payload id: ${fileId} is invalid`)
```

Comment:
Lets update this description to state that "no fileId was supplied" as supplying a fileId of `somethingRandom12345` is considered invalid as its not an ObjectId.